legend-depot-core-services/src/main/java/org/finos/legend/depot/services/projects/DependenciesCache.java

```diff
@@ -111,13 +111,13 @@ private Set<ProjectVersion> calculateTransitiveDependencies(ProjectVersion pv, F
             }
             else
             {
-                throw new RuntimeException(String.format("%s-%s-%s not found in store", pv.getGroupId(), pv.getArtifactId(), pv.getVersionId()));
+                throw new IllegalStateException(String.format("%s-%s-%s not found in store", pv.getGroupId(), pv.getArtifactId(), pv.getVersionId()));
             }
         }
         catch (Exception e)
         {
             LOGGER.error("error getting transitive dependencies {}",e.getMessage());
-            throw new RuntimeException(e);
+            throw new IllegalStateException(e);
```

Comment:
this is not an illegal state this is a runtime.. ie something not working when u calling prject etc